executors/cmdexec/brick.go

```diff
@@ -66,7 +66,7 @@ func (s *CmdExecutor) BrickCreate(host string,
 		fmt.Sprintf("mkfs.xfs -i size=512 -n size=8192 %v", devnode),
 
 		// Fstab
-		fmt.Sprintf("echo \"%v %v xfs rw,inode64,noatime,nouuid 1 2\" | tee -a %v > /dev/null ",
+		fmt.Sprintf("awk \"BEGIN {print \\\"%v %v xfs rw,inode64,noatime,nouuid 1 2\\\" >> \\\"%v\\\"}\"",
```

Comment:
Yeah. It uses a similar syntax to shell for "open this file in append mode".  The only thing that worries me here is that there's a lot of backslashes in there, but there's not a blocking concern, it's just annoying to parse mentally.

Reply:
I learnt after many attempts that single quotes can't be escaped using backslash inside other single quotes. However, double quotes can be. This seems to be the only way to do it.

Reply:
Ok. Good to know. Thanks!